indium-nodejs.el

```diff
@@ -113,6 +113,27 @@ If no process has been started, or if it was not started using
                 (read-from-minibuffer "Path: ")))
   (indium-nodejs--connect host port path))
 
+(defun indium-remove-leading-slash-from-uri (uri)
+  "Remove the leading slash from the given URI.
+NOTE: 'url-generic-parse-url' always return the path with a leading slash."
+  (substring uri 1 (length uri)))
+
+(defun indium-connect-nodejs-to-url (url)
+  "Connect to a node process with a given URL."
+  (let* ((parsed-url (url-generic-parse-url url))
+         (host (url-host parsed-url))
+         (port (or (url-port parsed-url) "9229"))
+         (path (indium-remove-leading-slash-from-uri
+                (car (url-path-and-query parsed-url)))))
+    (indium-nodejs--connect host port path)))
+
+;;;###autoload
+(defun indium-connect-nodejs-with-url (url)
+  "Connect to a node process with a given URL."
+  (interactive
+   (list (read-from-minibuffer "Url: " "")))
```

Comment:
Isn't the second argument to `read-from-minibuffer` useless here?

Reply:
Ah, ok. Thought it was required.